recipes/grpcio-tools/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.33.2" %}
+{% set name = "grpcio-tools" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: af40774c0275f5465f49fd92bfcd9831b19b013de4cc77b8fb38aea76fa6dce3
+  patches:
+    - 0001-Monkey-patch-distutils.ccompiler.spawn-to-elide-std-.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - grpcio >=1.33.2
```

Comment:
should this maybe pin to the same version as grpcio-tools? should we use `=={{ version }}` here?

Reply:
Yes, I thought about it but kept the same as in the Anaconda recipe. Let's do that!